site/content/entry/yuzu-progress-report-dec-2020/index.md

```diff
@@ -0,0 +1,160 @@
+
+## UI changes
+
+Finally, a very needed User Interface change, [language support](https://github.com/yuzu-emu/yuzu/pull/5239) is here. Thanks to work done by [Tobi](https://github.com/FearlessTobi) and many, *many* members of the community from around the globe, we can now offer support for 11 different languages, plus regional variations. ¡Muchas gracias!
+
+{{< imgs
+	"./language.png| You can find this in Emulation > Configure > General > UI > Interface language"
+  >}} 
+
+Anyone is welcome to help [expand the available language list.](https://www.transifex.com/yuzu-emulator/yuzu)
+
+For our command-line or shortcut lovers, [Morph](https://github.com/Morph1984) adds [command-line arguments.](https://github.com/yuzu-emu/yuzu/pull/5229) Current options are:
+
+- `yuzu.exe "path_to_game"` - Launches a game at `path_to_game`.
+- `yuzu.exe -f` - Launches the next game in fullscreen.
+- `yuzu.exe -g "path_to_game"` - Launches a game at `path_to_game`.
+- `yuzu.exe -f -g "path_to_game"` - Launches a game at `path_to_game` in fullscreen.
+
+Feel free to create desktop shortcuts of all of your games!
+
+Thinking of our fast typers, [toastUnlimited](https://github.com/lat9nq) added several [menubar access hot-keys](https://github.com/yuzu-emu/yuzu/pull/5223). Now, for example, an user can press `Alt + F, R, 2` to load the second most played game. Look Ma! No mouse!
+
+[german77](https://github.com/german77) added the option to [resize yuzu’s window to 1920x1080](https://github.com/yuzu-emu/yuzu/pull/5178), on top of the traditional 1280x720. This ensures a perfect 1:1 pixel ratio with native 1080p games while playing windowed, for those with high resolution displays.
+
+{{< imgs
+	"./resize.png| "
+  >}} 
+
+## Input changes
+
+[german77](https://github.com/yuzu-emu/yuzu/pull/5178) is back in action with a couple of input improvements.
+
+Adding a way to invert an analog stick axis was a problem we didn’t expect to have on Xbox and PlayStation controllers, but some alternative brands don’t seem to follow the specifications very closely. [Now if you right click on an axis, you get the option to invert it.](https://github.com/yuzu-emu/yuzu/pull/5233) 
+
+{{< imgs
+	"./invert.png| "
+  >}} 
+
+This is also a great way to invert a camera if a game doesn’t allow it in its own settings, for example in `Star Wars Jedi Knight II: Jedi Outcast`.
+
+Analog triggers were sometimes mapped inverted to what the user intended. As a way to avoid some bad moments to our users, now [the program takes two samples when mapping an analog trigger,](https://github.com/yuzu-emu/yuzu/pull/5265) to better determine the direction of movement. This was [ported from Citra](https://github.com/citra-emu/citra/pull/5509), so thanks guys!
```

Comment:
```suggestion
Analog triggers were sometimes mapped inverted to what the user intended. As a way to avoid some bad moments to our users, now [the program takes two samples when mapping an analog trigger,](https://github.com/yuzu-emu/yuzu/pull/5265) to better determine the direction of movement. This was [ported from Citra](https://github.com/citra-emu/citra/pull/5509), so thanks, guys!
```